### ISSUE TITLE
Subgraph is not working.

### ISSUE THREAD

**Issue by fhrryDeveloper** [QuickSwap/QuickSwap-subgraph#5]
https://thegraph.com/hosted-service/subgraph/learnforcode/quick-swap

Query: 
```
{
  uniswapFactories(first: 5) {
    id
    pairCount
    totalVolumeUSD
    totalVolumeETH
  }
  tokens(first: 5) {
    id
    symbol
    name
    decimals
  }
}
```
Result:
```
{
  "errors": [
    {
      "message": "indexing_error"
    }
  ]
}
```

**Comment by creazy231** [comment]
same over here

**Comment by fess-v** [comment]
same! https://thegraph.com/hosted-service/subgraph/sameepsi/quickswap06

**Comment by noescape00** [comment]
Anyone managed to find a fix for it?

**Comment by FriskyHamTitz** [comment]
Currently we don't manage those implementations of the graph and we use an external provider.

Have you tried deploying your own graph?

**Comment by noescape00** [comment]
> Currently we don't manage those implementations of the graph and we use an external provider.
> 
> Have you tried deploying your own graph?

No. Since https://thegraph.com/hosted-service/subgraph/sameepsi/quickswap03 and https://thegraph.com/hosted-service/subgraph/sameepsi/quickswap06 failed on exactly the same block I assume there is a bug in the indexer. 

Could you please tell which external provider you use?

**Comment by FriskyHamTitz** [comment]
Hmm, apparent the graph updated something in their side.  If you need this to work now you can check the change log and fix it.

Otherwise you will have to wait until we have an opportunity to fix this

**Comment by adedade** [comment]
Bump, is anyone deploying this ?
